### DISCUSSION TITLE
Access theme outside of components without caveats?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#4852] [Q&A] [answered]
Hi.
A like how now docs say that I safely can access `theme` outside components.
My question is - it is safe **to use it anywhere** as a replacement for useMantineTheme approach?
In v6 usage of `useMantineTheme` hook in some cases, according to user feedback, introduced problems with unnecessary re-renders.
So, I would like to not use it anymore, if this has no caveats at all.

**Comment by rtivital** [comment, marked as answer]
There are no caveats, if your theme does not contain any dynamic values, you can access it anywhere in your application
